### PR TITLE
:seedling: Enable golanci-lint for test/ submodule

### PR DESCRIPTION
**Pull request by vincepri**
Signed-off-by: Vince Prignano <vincepri@vmware.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -221,7 +221,8 @@ e2e-framework: ## Builds the CAPI e2e framework
 
 .PHONY: lint
 lint: $(GOLANGCI_LINT) ## Lint codebase
-	$(GOLANGCI_LINT) run -v $(GOLANGCI_LINT_EXTRA_ARGS)
+	# $(GOLANGCI_LINT) run -v $(GOLANGCI_LINT_EXTRA_ARGS)
```
Comment (stmcginnis) [inline Makefile:224]: Looks like you left this commented out for core from testing?

Comment (vincepri) [reply Makefile]: image

**Pull request merged**